notebooks/Tutorial9-Multitable.py

```diff
@@ -0,0 +1,521 @@
+        link_method="bottom_projection",
+        parent_link_key="doctor_id",
+        child_link_key="doctor_id",
+    ),
+]
+
+avat_parameters = [
+    BaseAvatarizationParameters(dataset_id=dataset_patient.id, k=5),
+    BaseAvatarizationParameters(dataset_id=dataset_doctor.id, k=5),
+    BaseAvatarizationParameters(dataset_id=dataset_visit.id, k=5),
+]
+# -
+
+# ## Anonymization
+
+# +
+
+avat_job = client.jobs.create_avatarization_multi_table_job(
+    AvatarizationMultiTableJobCreate(
+        parameters=AvatarizationMultiTableParameters(
+            table_links=table_links, table_parameters=avat_parameters
+        )
+    )
+)
+
+avat_job = client.jobs.get_avatarization_multi_table_job(avat_job.id, timeout=1000)
+results = avat_job.result.datasets
+avatars_df = {}
+for i, result in enumerate(results):
+    avatars = client.pandas_integration.download_dataframe(
+        result.avatars_dataset.id, should_stream=False
+    )
+    avatars_df[i] = avatars
+
+# -
+
+patient_avatar = avatars_df[0]
+patient_avatar.head()
+
+doctor_avatar = avatars_df[1]
+doctor_avatar.head()
+
+visit_avatar = avatars_df[2]
+visit_avatar.head()
+
+# ## Privacy metric computation
+# Similarly to multitable avatarization, privacy metrics calculation requires the specification of one set of parameter per table.
+
+# +
+privacy_parameters = parameters = [
+    BasePrivacyMetricsParameters(
+        original_id=dataset_patient.id,
+        unshuffled_avatars_id=results[0].sensitive_unshuffled_avatars_datasets.id,
+    ),
+    BasePrivacyMetricsParameters(
+        original_id=dataset_doctor.id,
+        unshuffled_avatars_id=results[1].sensitive_unshuffled_avatars_datasets.id,
+    ),
+    BasePrivacyMetricsParameters(
+        original_id=dataset_visit.id,
+        unshuffled_avatars_id=results[2].sensitive_unshuffled_avatars_datasets.id,
+    ),
+]
+
+privacy_job = client.jobs.create_privacy_metrics_multi_table_job(
+    PrivacyMetricsMultiTableJobCreate(
+        parameters=PrivacyMetricsMultiTableParameters(
+            table_links=table_links,
+            table_parameters=privacy_parameters,
+        )
+    )
+)
+
+privacy_job = client.jobs.get_privacy_metrics_multi_table_job(
+    privacy_job.id, timeout=1000
+)
+
+# -
+
+# ## Privacy metric results
+# The privacy metrics results are computed on multiple tables to verify as many attack scenario as possible.
+#
+# We define 4 types of multi table privacy scenario. For each scenario, we compute the same privacy metrics used with tabular data and further described [here](https://docs.octopize.io/docs/understanding/Privacy) by comparing orginal to avatars.
+#
+# 1. **STANDALONE**:
+#     Privacy metrics are applied on original unmodified tables at individual level. (patient and doctor in our example)
+#
+# 2. **TO_TOP_ENRICHED**:
+#     Privacy metrics are applied on individual level tables (patient and doctor) enriched with aggregated information from successor tables (visit).
+#
+# 3. **TO_BOTTOM**:
```

Comment:
```suggestion
# 3. **TO_BOTTOM_ID_PROPAGATED**:
```
if we want to have the exact naming as `PrivacyMetricsComputationType`, but that can be discussed if we want that